stdlib/src/utils/__init__.mojo

```diff
@@ -13,11 +13,11 @@
 """Implements the utils package."""
 
 from .index import Index, IndexList, product
-from .inline_string import InlineString
+from collections.string.inline_string import InlineString
```

Comment:
**Question** Do you plan on removing this re-export from `utils` in favor of doing it in `collections/string/__init__.mojo`?  Then we can fix the imports everywhere.